tests/validate/aie_pl_test/src/host.cpp

```diff
@@ -176,8 +181,18 @@ main(int argc, char* argv[])
     std::cout << "start pl controller kernel" << std::endl;
     // start input kernels
 
+    // start sender_receiver kernels
+    auto sender_receiver_r1 = xrt::run(sender_receiver_k1);
+    sender_receiver_r1.set_arg(0, num_iter);
+    sender_receiver_r1.set_arg(1, in_bo1);
+    sender_receiver_r1.set_arg(2, out_bo1);
+    sender_receiver_r1.start();
+    std::cout << " start sender-receiver kernel" << std::endl;
+
     controller_r1.wait();
     std::cout << " pl controller wait complete" << std::endl;
+    sender_receiver_r1.wait();
+    std::cout << " sender_receiver wait complete" << std::endl;
```

Comment:
Was this what was causing the hang on the second run? The sender_receiver was never collected?

Reply:
This change is not needed.  Only the PL controller changes are needed.
Let me revert this change.